src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1142,43 +1142,23 @@
     <Field Name="InitializerExpressionOpt" Type="BoundExpression" Null="allow"/>
   </Node>
   
-
-  <!-- 
-  Tuple literals can exist in two forms - literal and converted literal.
-  This is the base node for both forms.
-  -->
-  <AbstractNode Name="BoundTupleExpression" Base="BoundExpression">
-    <Field Name="Arguments" Type="ImmutableArray&lt;BoundExpression&gt;"/>
-  </AbstractNode>
-
   <!-- 
   Tuple literals can convert to the target type.
   Once converted to a target type, they cannot be target-typed again. 
   The tuple literal is one which has not been converted to a target type.
   -->
-  <Node Name="BoundTupleLiteral" Base="BoundTupleExpression">
+  <Node Name="BoundTupleLiteral" Base="BoundExpression">
```

Comment:
Previously we had target-typed and not target-typed literals represented by distinct nodes. The purpose was -
* make it impossible to target-type the same tuple literal twice
* trivially catch bugs if unconverted literals reach the emit phase
* reduce the size of bound nodes to minimally needed (converted tuple does not need to keep names, unconverted does not need two types)

That was similar to bound and unbound lambda nodes.

The new node combines it all together into one node.

Is it possible to keep converted and unconverted tuple nodes distinct?